geode-book/master_middleman/source/subnavs/geode-subnav.erb

```diff
@@ -2189,25 +2189,25 @@ gfsh</a>
                         </ul>
                     </li>
                     <li class="has_submenu">
-                        <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html">Redis API for <%=vars.product_name%></a>
+                        <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible-with-redis.html"><%=vars.product_name%> APIs Compatible with Redis</a>
                         <ul>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#using-the-api">Using the Redis API for <%=vars.product_name%></a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible-with-redis.html#using-the-api">Using <%=vars.product_name%> APIs Compatible with Redis</a>
                             </li>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#included-commands">Included Redis Commands</a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible-with-redis.html#included-commands">Included Redis Commands</a>
```

Comment:
Should this be "supported" instead of "included"?

Reply:
good catch, this way was broken